### ISSUE TITLE
Create predict_last_forecast equivalent to plot_last_forecast (Get results as csv after forecasting with n_lags)

### ISSUE THREAD

**Issue by artem1480** [ourownstory/neural_prophet#318]
How to get result? y cloumn and yhat are NAN
![image](https://user-images.githubusercontent.com/60170640/116005492-dac50f80-a60f-11eb-9081-229a2baccb08.png)

**Comment by artem1480** [comment]
![image](https://user-images.githubusercontent.com/60170640/116005544-1cee5100-a610-11eb-9882-6b1be2b3f588.png)

**Comment by artem1480** [comment]
![IMG_20210425_223933.jpg](https://user-images.githubusercontent.com/60170640/116007184-24fdbf00-a617-11eb-8af8-40f6b051e03e.jpg)

**Comment by Shiivong27** [comment]
If you're using n_forecasts = 10 and n_lags = 10 (for e.g.), then when you are making your future dataframe with the 'make_future_dataframe' method, be sure to define 'df' as your training set, 'regresors_df' as your test set, 'periods' as the length of the test set and set 'n_historic_predictions' to true.

When you forecast on the test dataset, you'll end up with predictions ranging from yhat1 to yhat10 (since n_forecasts is 10) with most of them being None. Plot the yhat10 values as they'll be the latest predictions of the model.

Hope that helps.

**Comment by ourownstory** [comment]
Thank you @Shiivong27 

@artem1480 might you be looking for a version of plot_last_forecast that returns the df with the last predictions instead?

**Comment by artem1480** [comment]
> Thank you @Shiivong27 
> 
> @artem1480 might you be looking for a version of plot_last_forecast that returns the df with the last predictions instead?

Perhaps. Where i can get it?

**Comment by ourownstory** [comment]
> > Thank you @Shiivong27
> > @artem1480 might you be looking for a version of plot_last_forecast that returns the df with the last predictions instead?
> 
> Perhaps. Where i can get it?
It currently does not exist. Would you like to solve this issue by contributing it? 
You could use the plot_last_forecast as base code and just remove the plotting and return the forecast instead.

**Comment by artem1480** [comment]
> > > Thank you @Shiivong27
> > > @artem1480 might you be looking for a version of plot_last_forecast that returns the df with the last predictions instead?
> > 
> > Perhaps. Where i can get it?
> 
> It currently does not exist. Would you like to solve this issue by contributing it?  :)
> You could use the plot_last_forecast as base code and just remove the plotting and return the forecast instead.
> 

Ok. I'll try.

**Comment by ourownstory** [comment]
mostly addressed in #701
